### ISSUE TITLE
Icons render wrong when you zoom in and out on Chrome

### ISSUE THREAD

**Issue by jordanellis** [nostalgic-css/NES.css#427]
**Describe the bug**
When you zoom in on a page that has icons from nes.css on it, the icons begin to show with white grid lines in between them as if the squares used to make them aren't touching anymore.

**To Reproduce**
1. Go to https://nostalgic-css.github.io/NES.css/ and view the icons.
2. Zoom in (and out at different percentages) on the browser.
3. View icons rendered with gaps.

**Expected behavior**
Icons should be rendered similar to how they get rendered at 100% zoom.
![Screen Shot 2020-06-02 at 5 15 19 PM](https://user-images.githubusercontent.com/20527219/83570907-d7ef9680-a4f4-11ea-9dc5-fe1f3ef58e60.png)

**Environment:**
 - OS: Mac
 - Browser: Chrome
 - Build environment (i.e. NodeJS): N/A

**Suggestion(s) for fixing this issue**
Unfortunately I don't know much about rendering graphic

**Additional context**
Love NES.css!

**Comment by SecretBase** [comment]
I think that kind of rendering bug on the browser when using box shadow as a pixel art

https://codepen.io/louflan/pen/mdJKRWN

I tried others people pixel art using same box shadow trick. also having the same problems.

I guess that just some weird browsers render bug. IMHO there nothing a framework can do about it. unless change those pixel art not using the box shadow trick.

Mac chrome at 110% zoom level
![image](https://user-images.githubusercontent.com/5130999/83639259-0648a100-a5dd-11ea-80c1-3ea89349db74.png)

Mac chrome at 67% zoom level
![image](https://user-images.githubusercontent.com/5130999/83639366-2b3d1400-a5dd-11ea-870f-3e4e2c08a50b.png)

Mac Safari zoom out 1 level
![image](https://user-images.githubusercontent.com/5130999/83639545-69d2ce80-a5dd-11ea-91c2-4678941c7de6.png)

Windows 10 Chrome 90% zoom level

![image](https://user-images.githubusercontent.com/5130999/83639809-ccc46580-a5dd-11ea-8493-e15eb49bc0dc.png)

**Comment by guastallaigor** [comment]
> I guess that just some weird browsers render bug. IMHO there nothing a framework can do about it. unless change those pixel art not using the box shadow trick.

Yes, that's exactly what I thought. I saw this bug before, but I couldn't do anything about it.
That's why I put the "help wanted" label :smile:

**Comment by HiKaylum** [comment]
May be possible converting these to SVGs rather than using box-shadow but will have to look into it and would be a breaking change.

**Comment by hoangph271** [comment]
Not only icons but buttons are broken on Chrome as well...!  
<img width="342" alt="Screen Shot 2021-10-06 at 18 43 58" src="https://user-images.githubusercontent.com/39024711/136197303-8bbed132-9366-4bff-8e8b-17fd6bdc63ea.png">

Works well on Firefox...!
<img width="342" alt="Screen Shot 2021-10-06 at 18 44 08" src="https://user-images.githubusercontent.com/39024711/136197367-08803d27-823b-42d2-86cb-3b7d83934941.png">
